core-libs/setup/prerender/providers/prerender-providers.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { inject, StaticProvider } from '@angular/core';
+import { INITIAL_CONFIG } from '@angular/platform-server';
+import { SERVER_REQUEST_ORIGIN, SERVER_REQUEST_URL } from '@spartacus/core';
+
+/**
+ * Returns the providers for the prerender.
+ */
+export function getPrerenderProviders(options: {
+  requestOrigin: string;
```

Comment:
Let's add JSDoc - so customers know what to put here, with an example `https://example.com` (with ending slash? without ending slash?)